docs/Changelog.md

```diff
@@ -18939,6 +18939,49 @@ This version of the operator has been available since version 14 of the default
 <dd>Constrain seq_lens to integer tensor.</dd>
 </dl>
 
+### <a name="GridSampler-14"></a>**GridSampler-14**</a>
+
+  Given an input and a flow-field grid, computes the output using input values and pixel locations from grid.
+  Currently, only spatial (4-D) input are supported.
+
+#### Version
+
+This version of the operator has been available since version 14 of the default ONNX operator set.
+
+#### Attributes
+
+<dl>
+<dt><tt>align_corners</tt> : int (default is 0)</dt>
+<dd>If align_corners=1, the extrema (-1 and 1) are considered as referring to the center points of the input's corner pixels. If align_corners=0, they are instead considered as referring to the corner points of the input's corner pixels, making the sampling more resolution agnostic.</dd>
+<dt><tt>mode</tt> : string (default is bilinear)</dt>
+<dd>Three interpolation modes: bilinear (default), nearest and bicubic.</dd>
+<dt><tt>padding_mode</tt> : string (default is zeros)</dt>
+<dd>Supported Padding modes for outside grid values: `zeros`(default), `border`, `reflection`</dd>
+</dl>
+
+#### Inputs
+
+<dl>
+<dt><tt>X</tt> (differentiable) : T</dt>
+<dd>4-D tensor of shape (N, C, inH, inW), where N is the batch size, C is the numbers of channels, inH and inW are the height and width of the data.</dd>
+<dt><tt>grid</tt> (non-differentiable) : T</dt>
+<dd>4-D tensor of shape (N, outH, outW, 2), where outH and outW is the height and width of offset and output.</dd>
```

Comment:
Can you add details around possible values in grid, and how should the out of range values be treated

Reply:
Thanks, added.